src/main/groovy/edu/oregonstate/mist/personsapi/PersonsApplication.groovy

```diff
@@ -23,19 +24,26 @@ class PersonsApplication extends Application<PersonsApplicationConfiguration> {
         this.setup(configuration, environment)
 
         DBIFactory factory = new DBIFactory()
-        DBI readJdbi = factory.build(environment, configuration.getReadDataSourceFactory(),
-                "readJdbi")
-        PersonsDAO personsDAO = readJdbi.onDemand(PersonsDAO.class)
-        PersonsStringTemplateDAO personsStringTemplateDAO = readJdbi.onDemand(
+        DBI bannerReadJdbi = factory.build(
+                environment, configuration.getBannerReadDataSourceFactory(), "bannerReadJdbi")
+        BannerPersonsReadDAO bannerPersonsReadDAO = bannerReadJdbi.onDemand(
+                BannerPersonsReadDAO.class)
+        PersonsStringTemplateDAO personsStringTemplateDAO = bannerReadJdbi.onDemand(
                 PersonsStringTemplateDAO.class)
 
-        DBI writeJdbi = factory.build(
-                environment, configuration.getWriteDataSourceFactory(), "writeJdbi")
-        PersonsWriteDAO personsWriteDAO = writeJdbi.onDemand(PersonsWriteDAO.class)
+        DBI bannerWriteJdbi = factory.build(
+                environment, configuration.getBannerWriteDataSourceFactory(), "bannerWriteJdbi")
+        BannerPersonsWriteDAO bannerPersonsWriteDAO = bannerWriteJdbi.onDemand(
+                BannerPersonsWriteDAO.class)
+
+        DBI odsReadJdbi = factory.build(
+                environment, configuration.getODSReadDataSourceFactory(), "odsReadJdbi")
+        ODSPersonsReadDAO odsPersonsReadDAO = odsReadJdbi.onDemand(
+                ODSPersonsReadDAO.class)
 
         environment.jersey().register(
-            new PersonsResource(personsDAO, personsStringTemplateDAO,
-                    personsWriteDAO, configuration.api.endpointUri)
+            new PersonsResource(bannerPersonsReadDAO, personsStringTemplateDAO,
+                    bannerPersonsWriteDAO, odsPersonsReadDAO, configuration.api.endpointUri)
```

Comment:
is the `odsPersonsReadDAO` supposed to be here?

Reply:
@Ruefa yes, I modified the `PersonResource` class to take `odsPersonReadDAO`: https://github.com/osu-mist/persons-api/pull/65/files#diff-ae9b2e6fcd32d2b3ce89dabe75276fc7R64